### PR TITLE
Migration to django-popolo part 2

### PR DESCRIPTION
**Pull request by mhl**
This is the second part of: #1181 for review. See that pull request for the context for this pull request.

This removes the old popit-django model references.

### CODE REVIEW

File: nuntium/migrations/0003_denull_popolo_people.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='answer',
+            name='popolo_person',
+            field=models.ForeignKey(to='popolo.Person'),
+            preserve_default=True,
+        ),
+        migrations.AlterField(
```
Comment (mhl) [inline nuntium/migrations/0003_denull_popolo_people.py]: Oops, this `AlterField` and the next one are spurious - they were accidentally generated by having the old models file present after 1a0c81a4e78 and running makemigrations, I'll add a fixup to remove them.

Comment (mhl) [inline nuntium/migrations/0003_denull_popolo_people.py]: And actually the same spurious AlterFields got into: b6fbffba8104

File: contactos/models.py [hunk 2]
```diff
@@ -22,8 +21,7 @@ def __unicode__(self):
 class Contact(models.Model):
     """docstring for Contact"""
     contact_type = models.ForeignKey('ContactType')
-    person = models.ForeignKey(Person)
-    popolo_person = models.ForeignKey(PopoloPerson, null=True, blank=True)
+    person = models.ForeignKey(PopoloPerson)
```
Comment (chrismytton) [inline contactos/models.py]: I think we can import `PopoloPerson` as `Person` now that we've removed the reference to the `popit.models.Person`?

Comment (mhl) [inline contactos/models.py]: Good point, I'll do that.

**Pull request merged**